src/command_modules/azure-cli-appservice/azure/cli/command_modules/appservice/_help.py

```diff
@@ -1474,3 +1474,13 @@
 type: command
 short-summary: Run a specific triggered webjob hosted on a web app.
 """
+
+helps['webapp remote-connection'] = """
+    type: group
+    short-summary: Create a remote connection using a tcp tunnel to your web app
```

Comment:
suggest `manage remote connections to your web app`

Reply:
@patricklee2 given we are moving this to core from extension, most of the documentation needs to be updated. If we want to make any changes this would be the time to change it. 

//CC: @rramachand21

Reply:
Also once this releases on core - can you make sure to update the command on extension with a message to start using the core command & eventually we can remove it completely from the extension.

Reply:
I'll rename it to create-remote-connection, add a message to the extension and start changing documentation.